test/var_action_test/effr_calc.F90

```diff
@@ -23,7 +23,7 @@ subroutine effr_calc_run(ncol, nlev, effrr_in, effrl_inout, &
       real(kind_phys),    intent(in)    :: effrr_in(:,:)
       real(kind_phys),    intent(inout) :: effrl_inout(:,:)
       real(kind_phys),    intent(out)   :: effri_out(:,:)
-      real(kind_phys),    intent(inout) :: effrs_inout(:,:)
+      real(8),intent(inout) :: effrs_inout(:,:)
```

Comment:
Why? I think this should be reverted. Or is this something you want to test for explicitly (now seeing the change in the meta file)?

Reply:
This is needed to test the kind transformations.

Reply:
Ok, thanks!